fluent-plugin-kafka.gemspec

```diff
@@ -19,6 +19,7 @@ Gem::Specification.new do |gem|
   gem.add_dependency "fluentd", [">= 0.10.58", "< 2"]
   gem.add_dependency 'ltsv'
   gem.add_dependency 'ruby-kafka', '>= 1.4.0', '< 2'
+  gem.add_development_dependency "rdkafka", ">= 0.6.0"
```

Comment:
We shouldn't include rbkafka even if development dependency.
How about adding this in Gemfile instead?

Reply:
I've moved it to Gemfile and enable to switch by an environment variable `USE_RDKAFKA`.
BTW is there any obvious reason we it shouldn't be added to development dependency?

Reply:
> https://github.com/zendesk/ruby-kafka/pull/884#discussion_r568087727
> One of the ruby-kafka maintainer wants to be opt-in whenever C code included gem.

https://github.com/fluent/fluent-plugin-kafka/pull/405#discussion_r696322524

Reply:
It's reasonable for digest-murmurhash because it's used with ruby-kafka.
But I don't think reasoble for rdkafka, because it's not used with ruby-kafka.

Reply:
Anyway, rdkafka is optional for this plugin and I removed the dependency from gemspec.
Can I merge this?

Reply:
Yes. Go ahead, please.